tests/mitoai_ui_tests/mitoai.spec.ts

```diff
@@ -72,6 +72,80 @@ test.describe('Mito AI Chat', () => {
     await expect(page.locator('.cm-codeDiffInsertedStripe')).toBeVisible();
   });
 
+  test('Code diffs are automatically rejected before new messages are sent', async ({ page }) => {
+    await createAndRunNotebookWithCells(page, ['\nprint("cell 0")']);
```

Comment:
I think we can remove \n here since we're handling it in the createAndRunNotebookWithCells function